service/gcs/main.go

```diff
@@ -59,10 +58,6 @@ func main() {
 		logrus.Fatal(err)
 	}
 
-	if level == logrus.DebugLevel {
```

Comment:
I would prefer we just stop passing DebugLog level in init by default. That way we can still get debug logs in vsockexec if we pass this verbosity.